### ISSUE TITLE
Code assist should not put unnecessary quotes around string value

### ISSUE THREAD

**Issue by tedepstein** [RepreZen/KaiZen-OpenAPI-Editor#41]
I tried Ctrl+space after the `in` key at the end of this code snippet, and it immediately put in the quoted value `"body"`.  We should only use quotes where they are required.

```
paths:
  /products:
    get:
      summary: Product Types
      description: |
        The Products endpoint returns information about the *Uber* products
        offered at a given location. The response includes the display name
        and other details about each product, and lists the products in the
        proper display order.
      parameters:
        - name: latitude
          in: "body"
```

**Comment by tfesenko** [comment]
Still present in the product.

**Comment by ghillairet** [comment]
Double quotes have been removed from those proposals.

Also changed how the string proposals behavior, when the proposal is a string, it will not add empty double quotes when the user insert the proposal, and the display has been changed to this:

![screen shot 2016-01-08 at 16 38 40](https://cloud.githubusercontent.com/assets/148045/12201855/c968f2c6-b626-11e5-920b-1a27f16f5120.png)

Also added an empty proposal for when no solutions can be found.

![screen shot 2016-01-08 at 16 38 33](https://cloud.githubusercontent.com/assets/148045/12201865/d81b0f7a-b626-11e5-9a34-c65b007f8b41.png)

**Comment by tedepstein** [comment]
In progress.  Test failure and some issues noted in the pull request #63

**Comment by tedepstein** [comment]
@tfesenko , Guillaume has added a new commit last week.  Please have a look.

**Comment by tfesenko** [comment]
@tedepstein , I only see commit 8c12723 which was merged to the master as a part of PR #41 .
It works. Why the issue was not closed then? Probably I just forgot to close the issue.
